Application/EdFi.Ods.SandboxAdmin/Initialization/InitializationEngine.cs

```diff
@@ -163,7 +163,26 @@ public void RebuildSandboxes()
             List<string> resetSandboxKeys = new List<string>();
             try
             {
+                // The code analyzer wants to change to the following, which I do not like due to the
+                // "database style" of Linq. We should decide if we want to disable or keep this rule.
+                // More info: https://rules.sonarsource.com/csharp/RSPEC-3267
+#pragma warning disable S125 // Sections of code should not be commented out
+                /*
+                foreach (var (clientProfile, sandbox) in from user in _users
+                                                            let clientProfile = _clientAppRepo.GetUser(user.Value.Email)
+                                                            from sandbox in user.Value.Sandboxes.Where(x => x.Value.Refresh)
+                                                            select (clientProfile, sandbox))
+                {
+                    if (resetSandboxKeys.Contains(sandbox.Key))
+                    {
+                        continue;
+                    }
+                */
+#pragma warning restore S125 // Sections of code should not be commented out
+
+#pragma warning disable S3267 // Loops should be simplified with "LINQ" expressions
                 foreach (var user in _users)
+#pragma warning restore S3267 // Loops should be simplified with "LINQ" expressions
```

Comment:
Read above. I might actually like the code suggestion, if I had bothered to rewrite the Linq statement in the "proper" (preferred) way.

Reply:
I used to like the SQL-like syntax back in the day, but time has spoken and this style is rarely seen these days.